example/about.md

```diff
@@ -7,9 +7,9 @@ permalink: /about/
 This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](http://jekyllrb.com/)
 
 You can find the source code for the Jekyll new theme at:
-<span class="svg-icon github"></span> /
+<a href="https://github.com/{{ site.github_username }}"><span class="svg-icon github"></span><span class="username">{{ site.github_username }}</span></a> /
 [minima](https://github.com/jekyll/minima)
 
 You can find the source code for Jekyll at
-<span class="svg-icon github"></span> /
+<a href="https://github.com/{{ site.github_username }}"><span class="svg-icon github"></span><span class="username">{{ site.github_username }}</span></a> /
 [jekyll](https://github.com/jekyll/jekyll)
```

Comment:
`<a href="https://github.com/{{ site.github_username }}">[...]</a>` doesn't make sense here. It simply needs to point to the Jekyll organisation.

```
You can find the source code for the Jekyll new theme at:
<span class="svg-icon github"></span> [jekyll]((https://github.com/jekyll) /
[minima](https://github.com/jekyll/minima)
```
You may also take a cue from this [PR](https://github.com/jekyll/jekyll/pull/5903). The difference is I left out the icons entirely as part of the PR. I feel, its cleaner that way.